### DISCUSSION TITLE
Error Return type of Dallgoot\Yaml\Types\YamlObject::jsonSerialize()

### DISCUSSION THREAD

**Discussion by raicho** [dallgoot/yaml#15] [General]
Hi 

@deprecated Return type of Dallgoot\Yaml\Types\YamlObject::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice.
This issue needs to be addressed to ensure compatibility with future versions of PHP.

**Comment by dallgoot** [comment]
Hi :) 
thanks for the feedback. This is already the case in the last version : https://github.com/dallgoot/yaml/blob/master/src/Types/YamlObject.php
What version are you using ?
